components/drawer/content/testimonials.tsx

```diff
@@ -0,0 +1,101 @@
+import { useDispatch, useSelector } from 'react-redux'
+import { Button } from '@mui/material'
+
+import useForm from '@/utils/hooks/useForm'
+import FormValidations from '@/utils/validations/forms'
+import { TextField, RatingField } from '@/components/form'
+import { StyledFieldSet } from './StyledFormComponents'
+import { FormEvent, useState } from 'react'
+
+import { AppDispatch } from '@/lib/redux/store'
+
+const INITIAL_STATE = {
+    name: { value: '' },
+    email: { value: '' },
+    rating: { value: 0 },
+    review: {
+        value: ''
+    }
+}
+
+const {
+    validAlphaString,
+    validEmail,
+    validAlphaNumericSpacesSpecials,
+    validNonZeroNumber
+} = FormValidations
+
+const VALIDACHE = {
+    name: validAlphaString,
```

Comment:
`validAlphaNumericSpacesSpecials`. This should be our primary validation unless we specifically need otherwise

Reply:
Meaning for more general fields like this. We should only limit to alpha if we absolutely need to.

Reply:
i updated it to alllow spaces tho so that people can put their full name